test/test.js

```diff
@@ -18,8 +18,8 @@ it('Should run tests', function () {
     var reportStream = createReportOutStream();
 
     var ps = gulpTestCafe({
-        browsers:        ['chrome', 'firefox'],
-        reportOutStream: reportStream
+        browsers: ['chrome', 'firefox'],
+        reporter: { outStream: reportStream }
```

Comment:
you shouldn't rewrite this test to ensure that you didn't add a breaking change.